cmd/podman/machine/init.go

```diff
@@ -60,6 +60,28 @@ func init() {
 
 	_ = initCmd.RegisterFlagCompletionFunc(diskSizeFlagName, completion.AutocompleteNone)
 
+	extraDiskNumFlagName := "extra-disk-num"
+	flags.Uint64VarP(
+		&initOpts.ExtraDiskNum,
+		// TODO: change me once I commit this to https://github.com/containers/common/blob/main/pkg/config/config.go
+		// extraDiskNumFlagName, cfg.ContainersConfDefaultsRO.Machine.ExtraDiskNum,
+		extraDiskNumFlagName, "d", 0,
```

Comment:
I'd prefer to not use `-d` or `-s` as shorthand flags for these options, I think that the long flag should be sufficient

Reply:
Also, would prefer `--extra-disks` instead of `--extra-disk-num`, but naming is always hard

Reply:
I believe the shorthand flags are still relevant given the fact that the default flag is quite long to type. Or perhaps is there another concern with the letters I proposed?
Also, I think `--extra-disk-num` is more explanatory than `--extra-disks` at this point, but I agree naming is hard. I'd like to have more eyes on this before I address the change and avoid back and forth if possible :).